requirements.txt

```diff
@@ -30,20 +30,23 @@ lazy-object-proxy==1.3.1
 MarkupSafe==1.1.1
 mccabe==0.6.1
 mock==2.0.0
-pbr==5.1.3
 oauthlib==3.0.1
+pbr==5.1.3
+psycopg2==2.7.7
 psycopg2-binary==2.7.7
 pycodestyle==2.5.0
 pyflakes==2.1.0
 PyJWT==1.4.2
 pylint==2.3.0
 python-decouple==3.1
+python-http-client==3.1.0
 python-social-auth==0.3.6
 python3-openid==3.1.0
 pytz==2018.9
 requests==2.21.0
 requests-oauthlib==1.2.0
 ruamel.yaml==0.15.89
+sendgrid==5.6.0
```

Comment:
@BrianSerem, does this dependency contribute to this PR?

Reply:
I hope this has been resolved